pkg/products/cloudresources/reconciler.go

```diff
@@ -340,3 +347,58 @@ func overrideStrategyConfig(resourceType string, tier string, croStrategyConfig
 
 	return nil
 }
+
+// reconcileCIDRValue sets the CIDR value in the ConfigMap from the addon
```

Comment:
If possible we should also look to read in the notifications email address that will be added as part of https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/310/diffs#diff-content-ebae93872f6cae079f57ba32f871b9eaf446340c